drivers/stepper/Kconfig

```diff
@@ -24,17 +24,13 @@ config STEPPER_SHELL
 	help
 	  Enable stepper shell for testing.
 
-config STEP_DIR_STEPPER
-	bool
-	help
-	  Enable library used for step direction stepper drivers.
-
 comment "Stepper Drivers"
 
 # zephyr-keep-sorted-start
 rsource "Kconfig.fake"
 rsource "Kconfig.gpio"
 rsource "adi_tmc/Kconfig"
+rsource "step_dir/Kconfig"
```

Comment:
Nit: probably let's just keep this area only for drivers and place all the lib functionalities above `comment "Stepper Drivers"`

Reply:
True, done.